labelme/app.py

```diff
@@ -567,6 +573,26 @@ def __init__(
         )
         fill_drawing.trigger()
 
+        highlight_group_id = action(
```

Comment:
I meant. 
Having single one "checkbox" in menu "Highlight group_id". If enabled then center of dots (maybe lines too) will have one color for all labels with the same id...

Reply:
Changed! Yes it reduced a bit of code, thanks!

Reply:
Working on the feature that we discussed when there are separate colored contours for same label names and similar group_id at once...

Reply:
Reverted commit with  feature "Label name by contour color highlighting", since it looks unpractical for now!

Reply:
Added ability to refresh the current file after button "Highlight group_id" was pressed!